heron/config/src/yaml/conf/aurora/packing.yaml

```diff
@@ -1,2 +1,3 @@
 # packing algorithm for packing instances into containers
-heron.class.packing.algorithm:    com.twitter.heron.packing.roundrobin.RoundRobinPacking
+heron.class.packing.algorithm:    com.twitter.heron.packing.roundrobin.ResourceCompliantRRPacking
+heron.class.repacking.algorithm:  com.twitter.heron.packing.binpacking.FirstFitDecreasingPacking
```

Comment:
@huijunw - thanks for the PR - should we use the same algorithm for both?

Reply:
done